### ISSUE TITLE
gather module uses deprecated methods

### ISSUE THREAD

**Issue by beyefendi** [ChrisTruncer/Just-Metadata#37]
[>] Please enter a command: gather whois

```
/usr/local/lib/python2.7/dist-packages/ipwhois/net.py:138: UserWarning: allow_permutations has been deprecated and will be removed. It is no longer needed, due to the deprecation of asn_alts, and the addition of the asn_methods argument.
  warn('allow_permutations has been deprecated and will be removed. '
Traceback (most recent call last):
  File "JustMetadata.py", line 94, in <module>
    the_conductor.menu_system()
  File "/mnt/hgfs/kali-files/Just-Metadata/common/orchestra.py", line 256, in menu_system
    self.user_command.split()[1])
  File "/mnt/hgfs/kali-files/Just-Metadata/common/orchestra.py", line 381, in run_gather_command
    ig_obj.gather(self.system_objects)
  File "modules/intelgathering/whois_info.py", line 27, in gather
    incoming_ip_obj[0].ip_whois = ip_whois.lookup()
AttributeError: IPWhois instance has no attribute 'lookup'
```

**Comment by ChrisTruncer** [comment]
Yeah, I believe I do need to look into this.  Thanks for letting me know.

**Comment by ChrisTruncer** [comment]
Fixed this in #38. It shows a warning message, but I expect that to be removed once the IPWhois library is updated. Code works now though.
